components/Dropdown/Dropdown.less

```diff
@@ -37,6 +41,7 @@
   }
   .dropdown-hidden {
     display: none;
+    visibility: hidden;
```

Comment:
why display none and visibility hidden? Is your visibility transition working? slow it down and take a look/